### DISCUSSION TITLE
Ayuda con las prácticas de INTRO

### DISCUSSION THREAD

**Discussion by MarioRoque01** [Launch-X-Latam/MisionFrontEnd#322] [Q&A]
Buen dia a todos, intento hacer las prácticas de INTRO como por ejemplo la de requerimientos pero sinceramente ando algo perdido (si vi las clases), si alguien ya realizo estas prácticas, ¿podría dejarme ver sus prácticas de INTRO para poder guiarme y entender mejor? por favor!!

Se los agradeceria mucho.

**Comment by RIvanCF** [comment]
Date una vuelta por mi repo amigo, pero métele nitro, que si andas algo atrasado 
https://github.com/RIvanCF/LaunchXLATAM_S1

**Reply by MarioRoque01** [reply]
Muchas gracias amigo :D , acabe la práctica de HTML, sabes si para mañana se sube solo la práctica de HTML, o ¿que otras prácticas se suben?

**Reply by RIvanCF** [reply]
Se suben HTML y CSS

**Reply by MarcoAntonio159** [reply]
> Se suben HTML y CSS

Hola, 

Disculpa a donde se suben, las practicas? hay algun repositorio, o formulario en especifico?

Gracias!.

**Reply by AlbertoSF99** [reply]
En realidad, solo se sube la de Abogabot y HTML, la de CSS aún no hay fecha. Y lo subes a tu repositorio de Github.
